container/id/id.go

```diff
@@ -65,26 +69,11 @@ func (id *ID) Parse(s string) error {
 }
 
 // String returns base58 string representation of ID.
-func (id *ID) String() string {
-	return base58.Encode((*refs.ContainerID)(id).GetValue())
-}
-
-// Marshal marshals ID into a protobuf binary form.
-func (id *ID) Marshal() ([]byte, error) {
-	return (*refs.ContainerID)(id).StableMarshal(nil)
-}
-
-// Unmarshal unmarshals protobuf binary representation of ID.
-func (id *ID) Unmarshal(data []byte) error {
-	return (*refs.ContainerID)(id).Unmarshal(data)
-}
-
-// MarshalJSON encodes ID to protobuf JSON format.
-func (id *ID) MarshalJSON() ([]byte, error) {
-	return (*refs.ContainerID)(id).MarshalJSON()
-}
-
-// UnmarshalJSON decodes ID from protobuf JSON format.
-func (id *ID) UnmarshalJSON(data []byte) error {
-	return (*refs.ContainerID)(id).UnmarshalJSON(data)
+//
+// Zero ID has returns empty string.
+//
+// See also Parse.
+func (id ID) String() string {
```

Comment:
Lets replace it with `encoding.TestMarshaler` (same for `Parse`) and leave `fmt.Stringer` formatless.

Reply:
https://github.com/nspcc-dev/neofs-sdk-go/pull/186#discussion_r835918936